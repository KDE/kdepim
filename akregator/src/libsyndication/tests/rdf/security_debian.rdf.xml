<?xml version="1.0" encoding="iso-8859-1"?>
<?xml-stylesheet href="dsa-rdf.css" type="text/css"?>
<rdf:RDF
  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
  xmlns="http://purl.org/rss/1.0/"
  xmlns:dc="http://purl.org/dc/elements/1.1/"
  xml:lang="en"
>
<!-- source: http://security.debian.org/
     properties:
     - description: escaped HTML
     - dc:date: short date-only version
-->
<channel rdf:about="http://www.debian.org/security/dsa.rdf">
  <title>Debian Security</title>
  <link>http://security.debian.org/</link>
  <description>
Debian Security Advisories
  </description>
  <dc:date>2006-03-18T02:46:35+00:00</dc:date>
  <items>
    <rdf:Seq>
<rdf:li resource="http://www.debian.org/security/2006/dsa-1008" />
    </rdf:Seq>
  </items>
</channel>
<item rdf:about="http://www.debian.org/security/2006/dsa-1008">
  <title>DSA-1008 kdegraphics - buffer overflow</title>
  <link>http://www.debian.org/security/2006/dsa-1008</link>
  <description>
&lt;p&gt;Marcelo Ricardo Leitner noticed that the current patch in DSA 932
(&lt;a href=&quot;http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-3627&quot;&gt;CVE-2005-3627&lt;/a&gt;) for kpdf, the PDF viewer for KDE, does not fix all
buffer overflows, still allowing an attacker to execute arbitrary
code.&lt;/p&gt;
  </description>
  <dc:date>2006-03-17</dc:date>
</item>
</rdf:RDF>
