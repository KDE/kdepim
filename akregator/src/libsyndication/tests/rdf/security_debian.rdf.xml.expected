# Feed begin ######################
title: #Debian Security#
link: #http://security.debian.org/#
description: #<br/>Debian Security Advisories<br/>#
# Item begin ######################
id: #http://www.debian.org/security/2006/dsa-1008#
title: #DSA-1008 kdegraphics - buffer overflow#
link: #http://www.debian.org/security/2006/dsa-1008#
description: #<p>Marcelo Ricardo Leitner noticed that the current patch in DSA 932 (<a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-3627">CVE-2005-3627</a>) for kpdf, the PDF viewer for KDE, does not fix all buffer overflows, still allowing an attacker to execute arbitrary code.</p>#
datePublished: #Fri Mar 17 00:00:00 2006#
dateUpdated: #Fri Mar 17 00:00:00 2006#
# Item end ########################
# Feed end ########################

